cmd/validate-krew-manifest/main.go

```diff
@@ -40,7 +40,10 @@ var flManifest string
 
 func init() {
 	flag.StringVar(&flManifest, "manifest", "", "path to plugin manifest file")
-	flag.Set("logtostderr", "true") // Set glog default to stderr
+	// Set glog default to stderr
+	if err := flag.Set("logtostderr", "true"); err != nil {
+		glog.Fatal(err)
```

Comment:
I think it would be good to use fmt.Println + os.Exit here, as glog is currently not initialized probably.

Reply:
updated as suggested.